### PR TITLE
top level docs

### PR DESCRIPTION
**Pull request by jcs-odoo**

### CODE REVIEW

File: content/applications/finance/sign.rst [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+filled out with the user's detail if they are logged in.
+
+.. seealso::
+   - `Odoo Sign: product page <https://www.odoo.com/app/sign>`_
+
+Validity of electronic signatures
+=================================
+
+While electronic signatures are usable in most countries, we recommend you check the :doc:`validity
+of electronic signatures <sign/signature_validity>`.
```
Comment (AntoineVDV) [inline content/applications/finance/sign.rst:24]: Until that we add more content to this app's doc, don't you think that its two pages can be safely merged together?

**Review by AntoineVDV — APPROVED** [review]
This looks much nicer with (I hope) little effort! Good job :)

File: content/applications/finance/accounting.rst [hunk 2]
```diff
@@ -1,16 +1,169 @@
...
+:doc:`AI-powered invoice recognition <accounting/payables/supplier_bills/ocr>`,
+:doc:`synchronization with your bank accounts <accounting/bank/feeds/bank_synchronization>`, and
+:doc:`automatic suggestions for a simplified reconciliation process
+<accounting/bank/reconciliation/reconciliation_models>`.
+
+**Odoo Invoicing** is a standalone invoicing app that allows you to :doc:`issue invoices
+<accounting/receivables/customer_invoices/overview>`, send them to your customers, and manage
+payments, including :doc:`online payments
+<accounting/receivables/customer_payments/online_payment>`. It works as a "lighter" version of Odoo
+Accounting, and you can upgrade it to Odoo Accounting if you need more advanced accounting features.
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/applications/finance/accounting.rst:30]: @AntoineVDV 
It makes sense. I switch the order of these two, then

**Comment by jcs-odoo** [comment]
@AntoineVDV 
suggestions applied + added a third commit for "expenses"

**Review by AntoineVDV — APPROVED** [review]

**Pull request merged**